.github/workflows/main_to_prod.yml

```diff
@@ -0,0 +1,17 @@
+name: Main to Prod
+on:
+  schedule:
+    - cron: '0 0 * * *'
+jobs:
+  main_to_prod:
+    name: Create pull request
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Open PR
+        run: gh pr create --base prod --title "Production deployment" --body ""
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+        # @todo find exit codes when PR exists or no changes required.
+        continue-on-error: true
```

Comment:
It's just `1`.

```
a pull request for branch "main" into branch "prod" already exists:
https://github.com/simplytestme/website/pull/76
➜  simplytestme git:(main) ✗ echo $?
1
```

We could make a PHP script to execute this and grok the output. If "already exists" or "no changes", no error. Unless actual error.